### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by deining**
_Description of what this PR is changing or adding, and why:_

This PR fixes a few typos I spotted.

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/ui/layout/constraints.md [hunk 1]
```diff
@@ -1131,7 +1131,7 @@ class Example26 extends Example {
       'If all of Row\'s children are wrapped in Expanded widgets, each Expanded has a size proportional to its flex parameter, '
       'and only then each Expanded widget forces its child to have the Expanded\'s width.'
       '\n\n'
-      'In other words, Expanded ignores the preffered width of its children.';
+      'In other words, Expanded ignores the preferred width of its children.';
```
Comment (domesticmouse) [inline src/ui/layout/constraints.md:1134]: This change needs to also be made in the matching dart code to not trigger the excerpt CI

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: src/ui/layout/constraints.md (hunk 1 above)
Comment (parlough) [reply src/ui/layout/constraints.md:1134]: The matching Dart code line can be found at https://github.com/flutter/website/blob/263efceaf8a2e14c1332808983b1f70145975b3f/examples/layout/constraints/lib/main.dart#L1015

The `code-excerpt` instruction before this code block has the source filename, or you can run a global search on the adjusted to find these.

Comment (deining) [reply src/ui/layout/constraints.md:1134]: Thanks for helping me out here!
The issue is resolved now!

**Review by parlough — APPROVED** [review]
Thanks for these fixes! Looks great :)

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**